src/apps/material/material.entry.tsx

```diff
@@ -29,14 +30,19 @@ interface MaterialEntryTextProps {
   audienceText: string;
   fictionNonfictionText: string;
 }
+interface MaterialEntryUrlProps {
+  searchUrl: string;
+  materialUrl: string;
```

Comment:
Consider using [the `URL` type](https://developer.mozilla.org/en-US/docs/Web/API/URL) instead of raw strings.

It seems as if this may help with some type safety and improved value handling. I do not know if it works with Storybook though.